visualising-calibration-info/calibration_data.ipynb

```diff
@@ -61,11 +61,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Querying Calibration Data\n",
+    "\n",
     "Here we define a function that has 3 parameters. \n",
     "\n",
     "- client\n",
-    "- calibration_set_id: ID of the calibration set to use. This is optional and can be used to query previous results.\n",
-    "- filename: if provides, it specifies the file to save the response"
+    "- calibration_set_id: ID of the calibration set to use. This is optional and can be used to query previous results or calibration as at the time of a job\n",
```

Comment:
If the calibration set id is not provided it will query the latest calibration set